src/calculateRentalCost.js

```diff
@@ -28,7 +28,13 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 7) {
+    return days * 40 - 50;
+  } else if (days >= 3) {
```

Comment:
As in the 5th point of the checklist, it is written, you should change it, so it would look like this:
```
if (condition) {
  return x;
}

return y;
```

and not : 
```
if (condition) {
  return x;
} else {
  return y;
}
```

Reply:
Very good student review!